pygeoapi/provider/ogr.py

```diff
@@ -427,6 +427,10 @@ def _load_source_helper(self, source_type):
 
     def _get_next_feature(self, layer, feature_id):
         try:
+            if layer.GetFeatureCount() == 0:
```

Comment:
Should changes in this file be in another PR?

Reply:
They were not intended to be there, sorry. Also, those changes have been already merged